userspace/libscap/linux/scap_ppm_sc.c

```diff
@@ -1098,7 +1098,7 @@ ppm_sc_code scap_ppm_sc_from_name(const char *name) {
 	const char *sc_name = name;
 
 	if(name == NULL) {
-		return -1;
+		return PPM_SC_UNKNOWN;
```

Comment:
API change: we do not return `-1` (that is not a correct enum value); instead we return PPM_SC_UNKNOWN.
Note: this is what we already did in `scap_native_id_to_ppm_sc` below:
```
pm_sc_code scap_native_id_to_ppm_sc(int native_id) {
	if(native_id < 0 || native_id >= SYSCALL_TABLE_SIZE) {
		return PPM_SC_UNKNOWN;
	}
...
```